{{cookiecutter.project_slug}}/Dockerfile

```
@@ -4,12 +4,12 @@ FROM totvslabs/pycarol:latest
 FROM totvslabs/pycarol:{{ cookiecutter.pycarol_version }}
 {%- endif %}
 
+RUN pip install -r requirements.txt
+
 WORKDIR /app
 ADD requirements.txt /app/
 ADD {{cookiecutter.project_slug}}/ /app/{{cookiecutter.project_slug}}/
 
```

the pip install step is before copy requitements.txt step.
```suggestion

WORKDIR /app
ADD requirements.txt /app/
RUN pip install -r requirements.txt
ADD {{cookiecutter.project_slug}}/ /app/{{cookiecutter.project_slug}}/

```